src/components/DropZone/DropZone.tsx

```diff
@@ -35,7 +35,7 @@ type Type = 'file' | 'image';
 
 export interface DropZoneProps {
   /** Label for the file input */
-  label?: string;
+  label?: React.ReactNode;
```

Comment:
@Tom-Bonnike I might be really silly but should this say `React.ReactNode | string` or does `React.ReactNode` accept a string and render it out?

Reply:
`React.ReactNode` is anything that can be rendered by React:
```js
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Which is also why that change is non-breaking!